src/react-apps/applications/ux-editor/src/containers/Container.tsx

```diff
@@ -9,14 +9,25 @@ import { FormComponentWrapper } from '../components/FormComponent';
 import { SwitchComponent } from '../components/widget/SwitchComponent';
 import { makeGetDesignModeSelector } from '../selectors/getAppData';
 import { makeGetFormDataSelector } from '../selectors/getFormData';
-import { makeGetActiveFormContainer, makeGetLayoutComponentsSelector, makeGetLayoutContainerOrder, makeGetLayoutContainersSelector } from '../selectors/getLayoutData';
+import {
+  makeGetActiveFormContainer,
+  makeGetLayoutComponentsSelector,
+  makeGetLayoutContainerOrder,
+  makeGetLayoutContainersSelector,
+} from '../selectors/getLayoutData';
 import '../styles/index.css';
-
-import { Draggable, Droppable } from 'react-beautiful-dnd';
+import DroppableDraggableComponent from './DroppableDraggableComponent';
```

Comment:
It might still be better to use a wrapper here, so that we don't have to import these dependencies for runtime where we don't use drag-n-drop.